pycheribuild/projects/run_fvp.py

```diff
@@ -475,6 +484,14 @@ def setup_config_options(cls, **kwargs):
         cls.smp = cls.add_bool_option("smp", help="Simulate multiple CPU cores in the FVP", default=True)
         cls.force_headless = cls.add_bool_option("force-headless", default=False,
                                                  help="Force headless use of the FVP")
+        cls.fvp_gdbserver = cls.add_bool_option("gdbserver", default=False,
+                                                help="Enable FVP GDB debugging plugin and wait for GDB to connect")
+        cls.fvp_gdbport = cls.add_config_option("gdb-port", default=cls.default_gdb_port(), kind=int)
```

Comment:
Should we default to the same port as QEMU for consistency? I.e. 1234 by default but use `find_free_port()` in the same way as run_qemu.py?

Reply:
Good point, fixed.